pom.xml

```diff
@@ -96,5 +96,15 @@
             <scope>system</scope>
             <systemPath>${project.basedir}/lib/org.hl7.fhir.validator.jar</systemPath>
         </dependency>
+		<dependency>
```

Comment:
If we are doing all of our Jolt work on the repo this feeds into, can we back this out or are you using the JSON utils for anything?

Reply:
nevermind, looks like you are using jolt, we can address this later.

Reply:
Right. I use it for JSON parsing. Gives out a nice Map.